### DISCUSSION TITLE
Is there a way to delete labels for only opened issues?

### DISCUSSION THREAD

**Discussion by lyqht** [octokit/octokit.js#2338] [Q&A] [answered]
Context is that I would like to remove the `hacktoberfest` label from *open* issues since the event is over. However from the docs, it seems that the issues API only have a route for deleting labels from *all* issues/PRs.

I'm guessing a way to do it is to find all issues, look through each of them to see if they are open and then use the PATCH API to update the labels for each of them. This is a little manual, so I'm happy to hear if there's better alternatives!

**Comment by gr2m** [comment, marked as answer]
> I'm guessing a way to do it is to find all issues, look through each of them to see if they are open and then use the PATCH API to update the labels for each of them.

that's the way to go. If you have multiple repositories, it would make a good "remove labels from open issues" @octoherd script:
https://github.com/octoherd/octoherd

**Reply by lyqht** [reply]
Thanks for clarifying! Then i'll make a octoherd script for it.
